include/hip/channel_descriptor.h

```diff
@@ -28,12 +28,22 @@ THE SOFTWARE.
 // on NVCC path:
 
 
-#if (defined(__HIP_PLATFORM_HCC__) || defined(__HIP_PLATFORM_AMD__)) && !(defined(__HIP_PLATFORM_NVCC__) || defined(__HIP_PLATFORM_NVIDIA__))
+#if (defined(__HIP_PLATFORM_HCC__) || defined(__HIP_PLATFORM_AMD__)) &&                            \
+    !(defined(__HIP_PLATFORM_NVCC__) || defined(__HIP_PLATFORM_NVIDIA__)) &&                       \
+    !(defined(__HIP_PLATFORM_CLANG__) || defined(__HIP_PLATFORM_SPIRV__))
```

Comment:
What is the purpose of `__HIP_PLATFORM_CLANG__` and under what situation is it defined?

Reply:
We define it for all our paths. It's here to match the AMD and NVIDIA equivalents `__HIP_PLATFORM_HCC__` and `__HIP_PLATFORM_NVCC__`